core/components/_helpers/button-prop.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Button from '../atoms/button'
+export default (prop, overrides = {}) => {
```

Comment:
```diff
- prop
+ props
```

Reply:
It is actually one prop (either `primaryAction` or `secondaryAction` in the PageHeader case)

Reply:
then let's rename it to `action`

`prop` doesn't communicate the intent